contents/handbook/growth/marketing/index.md

```diff
@@ -71,18 +73,20 @@ But we think that the opportunity to build a strong brand off the back of these
+  - _Shareable content_ is stuff we think users and audiences outside PostHog will just find interesting. These may be technical deep dives or articles that give an insight into how PostHog works. We don't expect these to turn into signups, but they help us to build our unique and distinctive brand in the open source community. We share this content on 'super node' sites like Hacker News, Product Hunt and Reddit. 
+- [Product launches](https://posthog.com/handbook/growth/marketing/product-announcements). These bring transparency to the work that the PostHog team is doing. Product and engineering pull marketing here, not the other way around. Marketing's role is to look for cool things the team is already working on and to amplify them, often through a coordinated campaign that maximizes impact. Marketing's role is definitely _not_ to get teams to build features we think are 'marketable'. We define product broadly here - they can be anything from a major PostHog app feature to a cool internal tool that someone has built which we think the world should see. 
+  - We're currently experimenting with different types and size of launch to see what works most effectively here. 
+- [Paid ads](/handbook/growth/marketing/paid). This is intentionally at the bottom of the list. We advertise on Google Search and DuckDuckGo (via Bing) for _conversion_, and Twitter, LinkedIn, Reddit and Carbon for _awareness_. We use paid ads as a cheat code to accelerate learning, e.g. which SEO keywords work, testing out product positioning, as well as to ensure we are easily discoverable when people search for PostHog specifically ('branded keywords'). However, paids ads are expensive so our focus is always on organic growth first unless we find a paid channel that's wildly profitable. 
 
-Developer relations and community is also super important to us, but covered elsewhere as we don't strictly define it as part of marketing at PostHog. 
+[Developer relations and community](https://posthog.com/handbook/growth/developer-relations) is also super important to us, but covered elsewhere as we don't strictly define it as part of marketing at PostHog. Ditto for [Growth](https://posthog.com/handbook/growth/strategy). 
  
-The only [emails](/handbook/growth/marketing/newsletter) we send are to let people know about Product updates (see 'No sneaky shit') and we haven't yet figured out how we want to do [PR](/handbook/growth/marketing/press). 
+The only [emails](/handbook/growth/marketing/newsletter) we send are to let people know about Product updates (see 'No sneaky shit') and we haven't yet figured out how we want to do [PR](/handbook/growth/marketing/press), if at all. 
```

Comment:
```suggestion
The only [emails](/handbook/growth/marketing/newsletter) we send are to let people know about Product updates (see 'No sneaky shit') and we haven't yet figured out how we want to do [PR](/handbook/growth/marketing/press), if at all. 
```

We do actually have a newsletter with subscribers. However, we've never really sent it out.

If we do do this I'd push for:

- put _all_ the post content in the email (no sneaky shit + this means people actually read the content, which is its aim)
- it shouldn't be ultra frequent, and should be very carefully curated

Reply:
Yeah @andyvan-ph and I have been discussing this - we think we want to do _something_ in the future, but it's just not been a priority right now vs. other things we're working on.